### DISCUSSION TITLE
How I can configure so limits do not impact automergable prs

### DISCUSSION THREAD

**Discussion by ns-ggeorgiev** [renovatebot/renovate#16184] [Request Help] [answered]
Hi,

I am having a mono-repo with many owners. I expect that there will be multiple issues most of the time. And I also would like to have pr limits, else we talking about 3-4 hundred PRs.

At the same time I have automergable PRs. Right now they are kind of blocked from the limits and the other PRs. I see that there is exception for security issues to not honer the limits. Is there an option or this is a feature request that for automergable PRs the repo limits should not apply.

**Comment by rarkins** [comment, marked as answer]
I don't think there's currently a way to do this.

Relevant feature request: https://github.com/renovatebot/renovate/issues/10546
